Makefile

```diff
@@ -1,10 +1,30 @@
-all:
-	go get github.com/tools/godep
+CWD:=$(shell pwd)
+RUNC_TEST_IMAGE=runc_test
+PROJECT=github.com/opencontainers/runc
+TEST_DOCKERFILE=test_Dockerfile
+
+all: godep
 	godep go build -o runc .
```

Comment:
Should we document to but the godep binary directory in path or copy it to somewhere in the PATH.